src/app/containers/Ad/Amp/index.jsx

```diff
@@ -43,6 +43,8 @@ const constructAdJsonData = ({ service }) => {
 };
 
 const ampAdPropsMobile = ({ service }) => ({
+  'data-block-on-consent': 'default',
```

Comment:
Setting `data-block-on-consent` to be `default` fixes requests for ads being blocked till consent is either rejected or accepted. This is because it now follows default behaviour specified by defined policy object

Reply:
Got this from https://github.com/ampproject/amphtml/issues/15274#issuecomment-399247941